lib/http/middleware.go

```diff
@@ -166,9 +166,9 @@ func MiddlewareAuthCustom(fn CustomAuthFn, realm string, userFromContext bool) M
 var onlyOnceWarningAllowOrigin sync.Once
 
 // MiddlewareCORS instantiates middleware that handles basic CORS protections for rcd
-func MiddlewareCORS(allowOrigin string) Middleware {
+func MiddlewareCORS(cfg Config) Middleware {
```

Comment:
I would pass a pointer to Config here otherwise you are copying a big structure about.

Reply:
I did not know there were pointers in go. 🙈
Will check it out tomorrow.

Reply:
ok, that was easy. Should be fine now?